src/types.tsx

```diff
@@ -7,6 +7,7 @@ import {
   TextInputFocusEventData,
   ColorValue,
 } from 'react-native';
+import React from 'react';
```

Comment:
Can be cleaned up? On the diff it looks like this is the only line added to this source file, and this import is not used at all.

Reply:
As we're using `React` in types (for example for defining`React.ref`) this import cannot be removed 😕